>>> python -m pytest -vv issue_5_test.py

==================================================== test session starts =====================================================
platform win32 -- Python 3.12.0, pytest-7.4.3, pluggy-1.3.0 -- D:\Program Files\Python312\python.exe
cachedir: .pytest_cache
rootdir: D:\Avito\Python\Avito_Homeworks\Testing
plugins: cov-4.1.0
collected 8 items

issue_5_test.py::test_year_today PASSED                                                                                 [ 12%]
issue_5_test.py::test_get_year_2021_YYYY_MM_DD PASSED                                                                   [ 25%]
issue_5_test.py::test_get_year_1915_YYYY_MM_DD PASSED                                                                   [ 37%]
issue_5_test.py::test_get_year_1874_DD_MM_YYYY PASSED                                                                   [ 50%]
issue_5_test.py::test_get_year_3576_DD_MM_YYYY PASSED                                                                   [ 62%]
issue_5_test.py::test_get_exception_1 PASSED                                                                            [ 75%]
issue_5_test.py::test_get_exception_2 PASSED                                                                            [ 87%]
issue_5_test.py::test_get_exception_3 PASSED                                                                            [100%]

===================================================== 8 passed in 10.85s =====================================================